README.md

```diff
@@ -12,7 +12,14 @@ The primary user of object_store_ffi is the Julia package [RustyObjectStore.jl](
 
 #### Releasing
 
-New releases of object_store_ffi are made via GitHub releases.
+New releases of object_store_ffi are made by creating and pushing a tag with the new version (format: `v[0-9]+.[0-9]+.[0-9]+`).
+This will dispatch a release workflow that will build the assets and create the GitHub Release.
+
+```bash
+git tag -a v0.3.0 -m "New Release"
+git push --tags
+```
```

Comment:
maybe add that git tags / releases should be kept in-sync with the Cargo.toml `version` ?

Reply:
Added a note on the semver line